src/sbvr-api/permissions.coffee

```diff
@@ -122,7 +123,28 @@ exports.setup = (app, sbvrUtils) ->
 		PREPARSE: ({ req }) ->
 			apiKeyMiddleware(req)
 		POSTPARSE: ({ req, request }) ->
+			if (request.method == 'POST' and request.odataQuery.property?.resource == 'canAccess')
+				request.odataQuery.options = { '$select': { 'properties': [ { 'name': 'id' } ] } }
+				{ action, method } = request.values
+				if method? and methodPermissions[method]?
```

Comment:
I think it would make sense to return a `400` if both `action` and `method` are specified, rather than silently dropping the `action`, that way it avoids the possibility of confusion